### ISSUE TITLE
Error Status 0xc0000022 should be more informative

### ISSUE THREAD

**Issue by isaak654** [sandboxie-plus/Sandboxie#2931]
### Is your feature request related to a problem or use case?

Dialog error `Error Status 0xc0000022 (Access Denied)` should be more informative so as not to alarm GitHub users using test bulds:

<img alt="Signature file is missing (test builds)" src="https://github.com/sandboxie-plus/Sandboxie/assets/12372772/f64d4eb8-c2a3-41d3-8172-da267940be99" height="350px">

### Describe the solution you'd like

Add a further line explaining that a .sig file is missing.

**Comment by isaak654** [comment]
Unfortunately I still get the usual Error Status 0xc0000022, even after commit 8bca9c7a41502b09f7d9500cc0c150f2daf898d6

**Comment by offhub** [comment]
Did you try with the driver?

**Comment by DavidXanatos** [comment]
this requires the new driver as it changed the returned error code to be more apropriate of what happened.
A user may end up with an access denided due to other issues as well and it would be wrong to then blame it on the signature.

**Comment by isaak654** [comment]
> Did you try with the driver?

Yes, I did, but I think I can confirm it only after installing the final version 1.9.4.
